barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover_impl.hpp

```diff
@@ -144,20 +143,18 @@ void ProtogalaxyProver_<Flavor, NUM_KEYS>::update_target_sum_and_fold(
     // the lagrange coefficients between the accumulator and the incoming key.
     // TODO(https://github.com/AztecProtocol/barretenberg/issues/1417): make this swapping logic more robust.
     if (incoming->overflow_size > accumulator->overflow_size) {
-        std::swap(accumulator->proving_key.polynomials, incoming->proving_key.polynomials);   // swap the polys
```

Comment:
Previously had to swap 3 separate (but equivalent) "circuit size" parameters. sheesh. Now just 1